turbo_tests.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.metadata["source_code_uri"] = "https://github.com/serpapi/turbo_tests"
   spec.metadata["changelog_uri"] = "https://github.com/serpapi/turbo_tests/releases"
 
-  spec.required_ruby_version = ">= 2.7"
+  spec.required_ruby_version = ">= 3.0"
```

Comment:
Unfortunately, we can't drop Ruby 2.7 support because it's faster that 3.x (ref: https://news.ycombinator.com/item?id=38687229).

Reply:
Got it, I removed this change!